application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -665,9 +657,41 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
   }) // end PlainList extends View
 
   jQuery.fullCalendar.views.plainList = PlainList;
-  
 } // if the XWiki.widgets.ModalPopup  is defined
-  
+
+});
+
+/**
+ * Deleting an event
+ */
+require(['jquery'], function($) {
+  /**
+  * Events triggered before deleteEvents modal is shown: save the button that triggers
+  * the modal, close MoccaCalendarPopup, update the name of event on displayed text.
+  */
+  $(document).on('show.bs.modal', '#deleteEvents', function(event) {
+    var relatedTarget = $(event.relatedTarget);
+    $(this).data('relatedTarget', $(relatedTarget));
```

Comment:
``relatedTarget`` is already a jQuery object (because you wrap ``event.relatedTarget`` above) so you don't need to call ``$`` again here and below (in a couple of places). It doesn't hurt, but it doesn't help either.